### DISCUSSION TITLE
How can we facilitates manual tests?

### DISCUSSION THREAD

**Discussion by eltongarbin** [ocean-ds/ocean-android#117] [Q&A]
For web we have [storybook](https://ocean-ds.github.io/ocean-web/) wich is a great tool to play with components and see the final result online. 
Is there something similar for android? I think the @ocean-ds/designers will appreciate it!

**Comment by mariofelesdossantosjunior** [comment]
I was researching about it and I found this tool, you need to try to create a POC and validate.
https://medium.com/storybookjs/storybook-for-mobile-applications-97e3a229fb3c
https://github.com/storybookjs/native
https://storybookjs.github.io/native/@storybook/native-controls-example/index.html?path=/story/button--example
